.github/actions/setup-environment/action.yml

```diff
@@ -312,23 +317,40 @@ runs:
           echo "gp2-api-url=https://api-$PR_NUMBER.gp2.asap.science" >> $GITHUB_OUTPUT
 
           if [ "$CRN_CREATE_CONTENTFUL_PR_ENV" = true ]; then
-            echo "setting up dedicated PR env"
-            echo "crn-algolia-index=asap-hub_CI-$PR_NUMBER" >> $GITHUB_OUTPUT
-            echo "crn-analytics-algolia-index=crn-analytics_CI-$PR_NUMBER" >> $GITHUB_OUTPUT
+            echo "setting up dedicated Contentful PR env"
             echo "crn-contentful-env-id=crn-$PR_NUMBER" >> $GITHUB_OUTPUT
+
+            if [ "$CRN_CREATE_ALGOLIA_PR_ENV" = true ]; then
```

Comment:
This `if [ "$CRN_CREATE_ALGOLIA_PR_ENV" = true ]; then` seems to be duplicated here
<img width="932" alt="Screenshot 2024-09-27 at 09 03 33" src="https://github.com/user-attachments/assets/3b682c95-de8e-4d10-a996-49c9e612eff5">